hw/ip/kmac/data/kmac.hjson

```diff
@@ -625,17 +625,39 @@
       regwen: "CFG_REGWEN"
       fields: [
         { bits: "HashCntW-1:0"
-          name: "threshold"
-          desc: "Hash Threshold"
-        }
-        { bits: "HashCntW+15:16"
           name: "hash_cnt"
           desc: "Hash (KMAC) counter"
           swaccess: "ro"
           hwaccess: "hwo"
         }
       ]
-    } // R: ENTROPY_REFRESH
+    } // R: ENTROPY_REFRESH_HASH_CNT
+    {
+      name: "ENTROPY_REFRESH_THRESHOLD_SHADOWED"
```

Comment:
As far as I know (I may be wrong), you don't have to create _SHADOWED separately. just defining `shadowed: true` then the reggen will create two copies of the register.

Reply:
reggen complained about the naming. something like "shadowed: true declared, but name has no suffix _shadowed"
Additionally, I hat to split the register up into two, because `hash_cnt` has `hwaccess: "hwo"` which cannot be part of a shadow-register. and shadow subregisters are not supported (shadowed: true is not allowed in the fields part).

Reply:
Indeed we impose a naming convention on shadow registers, and the entire register needs to be declared as shadowed i.e., we cannot define shadow registers on individual register fields.

If I understand this correctly, the main reason for having to split this out into a separate register is the conflicting `hwaccess` for the `hash_cnt` field which needs to be `hwo`, which is incompatible with the shadow reg at the moment:

https://github.com/lowRISC/opentitan/blob/60b7743b7a83cd1cf237e85e0088ec4e88a7cb8f/util/reggen/field.py#L136-L139

So overall this LGTM.

Does this also need a corresponding DIF update?

Reply:
`util/make_new_dif.py  --mode=regen --only=autogen` doesn't show any chances and CI was also ok (normally CI complains about missing new difs)